<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PastConj</class>
 <widget class="QDialog" name="PastConj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Russian Past Tense Conjugation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="LineEditRU" name="rufemEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ruplurLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Plural</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="ruplurEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rutitleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Past Tense for </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="runuetEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="rumascEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rufemLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Feminine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rumascLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Masculine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="runuetLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nueter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBttn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="LineEditEN" name="femEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>780</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>English Past Tense for</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="plurEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mascLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Masculine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="femLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Feminine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="mascEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="nuetEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="plurLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Plural</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nuetLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nueter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="stemLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Verb Stem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="stemEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="verbLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Past Tense English Verb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="verbEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditRU</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineeditru</header>
  </customwidget>
  <customwidget>
   <class>LineEditEN</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineediten</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>stemEdit</tabstop>
  <tabstop>rumascEdit</tabstop>
  <tabstop>rufemEdit</tabstop>
  <tabstop>runuetEdit</tabstop>
  <tabstop>ruplurEdit</tabstop>
  <tabstop>verbEdit</tabstop>
  <tabstop>mascEdit</tabstop>
  <tabstop>femEdit</tabstop>
  <tabstop>nuetEdit</tabstop>
  <tabstop>plurEdit</tabstop>
  <tabstop>quitBttn</tabstop>
  <tabstop>backBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
